scripts/primir_quantification.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+"""Pri-miR quantification.
+
+This script creates the miRNA primary transcript counting table build upon four
+columns: the primary transcript name, the count, the 5' end relative extension
+and the 3' end relative extension. First, the SAM file will be traversed to
+create a dictionary that will have as keys the alignment names and their NH tag
+as value. Thereafter, and under the assumption that the BED file is sorted by
+the primary transcript name, the counting will be made. For each pri-miRNA, the
+counting will consist on the sum of each of its intersecting alignment; the
+contribution is computed as 1/NH. Finally, the pri-miRNA name in the BED file
+will be split to retrieve the actual pri-miRNA name and the relative 
+extensions.
+
+Usage:
+    primir_quantification.py --bed BED --sam SAM --outfile outputfile
+"""
+
+import argparse
+import os
+from pathlib import Path
+
+import pysam
+
+
+def parse_arguments():
+    """Command-line arguments parser."""
+    parser = argparse.ArgumentParser(
+        description="Script to quantify pri-miRs."
+        )
+    parser.add_argument(
+        '-v', '--version',
+        action='version',
+        version='%(prog)s 1.0',
+        help="Show program's version number and exit"
+    )
+    parser.add_argument(
+        '-b', '--bed',
+        help="Path to the extended pri-miR intersection BED file.",
+        type=Path,
+        required=True
+    )
+    parser.add_argument(
+        '-s', '--sam',
+        help="Path to the SAM file containing the intersecting alignments.",
+        type=Path,
+        required=True
+    )
```

Comment:
Could be positional, but guess - given that two inputs are required - I can live with them being "options"

Reply:
If you rather have them as positionals I can make them but I thought it would be easier to have them as options.